python/api_helpers/README.md

```diff
@@ -0,0 +1,8 @@
+# REST API helpers
```

Comment:
Can we make this our "official" Python bindings? What would it take to package them as such?

If so, we need to add a page in `feldera/docs` pointing to this library (or even to auto-generated docs).  I guess this could be part of the next PR, which will remove the old Python bindings.

Reply:
Just noticed the PR comment about removing `api_helpers`. While I'm in favor of making the REST API ultra-ergonomic, a native Python library still seems useful.

Reply:
One reason we can do without a native python library for now is that we get documentation for free from OpenAPI. This is what it currently looks like if I click the python option from our public API.

![image](https://github.com/feldera/feldera/assets/434651/f04f341b-596b-4cbf-932b-fb293d86f3cf)

Once we support `PUT`, the API becomes even simpler as Simon mentioned. It will leave compilation and pipeline start/stop as maybe requiring a simple helper at best, both of which can also be handled more declaratively.

Reply:
In the latest version, the PUT requests are now used without a helper library.